packages/desktop-client/src/components/schedules/ScheduleDetails.jsx

```diff
@@ -73,6 +72,8 @@ function updateScheduleConditions(schedule, fields) {
 }
 
 export function ScheduleDetails({ id, transaction }) {
+  const { t } = useTranslation();
```

Comment:
Can the error text on lines 51 and 55 ever be displayed normally to the user? My hunch is yes though I wasn't able to find a repro—in which case, maybe we should translate them?